packages/fxa-content-server/app/tests/spec/views/post_verify/secondary_email/add_secondary_email.js

```diff
@@ -209,6 +209,24 @@ describe('views/post_verify/secondary_email/add_secondary_email', () => {
           assert.equal(args[1], error);
         });
       });
+
+      describe('adding existing owned email', () => {
```

Comment:
You could probably add this to the functional tests [here](https://github.com/mozilla/fxa/blob/9ad167e5e014d659b39bcb3d9ac938ff38223234/packages/fxa-content-server/tests/functional/settings_secondary_emails.js#L221).